app/assets/stylesheets/base.css.scss

```diff
@@ -69,6 +69,14 @@ a {
     margin: 0 auto;
 }
 
+.hide-display {
```

Comment:
There's a bootstrap class for this, no need to create our own. http://getbootstrap.com/css/#helper-classes-show-hide